src/Browscap/Filter/CustomFilter.php

```diff
@@ -16,6 +16,11 @@ class CustomFilter implements FilterInterface
      */
     private $fields = [];
 
+    /**
+     * @var string
```

Comment:
Please revert all changes in this filter.

Add a FilterCollection which implements the FilterInterface where you may add multiple Filters. Please implement it similar to the WriterCollection.